Client/Frontend/Library/HistoryPanel/HistoryPanelViewModel.swift

```diff
@@ -3,7 +3,7 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
 import Foundation
-import Logger
+import Common
```

Comment:
So, in some places I'm seeing this, where we remove Logger and add Common, and in others where we remove Logger and don't add Common. What the reasoning behind those?

Reply:
Some already had Common imported.